bundles/org.openhab.binding.hapero/src/main/java/org/openhab/binding/hapero/internal/HaperoBindingConstants.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hapero.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link HaperoBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Daniel Walter - Initial contribution
+ */
+@NonNullByDefault
+public class HaperoBindingConstants {
+    public static final String BINDING_ID = "Hapero";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_BRIDGE = new ThingTypeUID(BINDING_ID, "haperoBridge");
+    public static final ThingTypeUID THING_TYPE_FURNACE = new ThingTypeUID(BINDING_ID, "furnace");
+    public static final ThingTypeUID THING_TYPE_BUFFER = new ThingTypeUID(BINDING_ID, "buffer");
+    public static final ThingTypeUID THING_TYPE_BOILER = new ThingTypeUID(BINDING_ID, "boiler");
+    public static final ThingTypeUID THING_TYPE_HEATING = new ThingTypeUID(BINDING_ID, "heatingCircuit");
+
+    // List of all Group ids
+    public static final String GROUP_TEMPERATURES = "temperatures";
+    public static final String GROUP_STATUS = "status";
+    public static final String GROUP_POWER = "power";
+    public static final String GROUP_AIR = "air";
+
+    // List of all Channel ids
+    public static final String CHANNEL_COMBUSTIONTEMPERATURE = "temperatures#combustionTemp";
```

Comment:
You do not need to put the group name in the channel name.

Reply:
Channel.getUID().getId() has the group in it. And since the groups are static anyway, i figured it would be less hassle to 
just put it there instead of dealing with it in the code:

`protected @Nullable State getStateForChannel(Channel channel, Device device) {
        State state = null;
        Device newDevice = null;
        Float value;
        String stringValue;
        Integer intValue;
        String channelId = channel.getUID().getId(); << Here i get the ID
        HaperoBridgeHandler bridgeHandler = null;
        Bridge bridge = getBridge();

        if (bridge != null) {
            bridgeHandler = (HaperoBridgeHandler) bridge.getHandler();
        }

        if (bridgeHandler != null) {
            switch (channelId) {
                case HaperoBindingConstants.CHANNEL_COMBUSTIONTEMPERATURE:` << And here i use it

Reply:
Yes and there also exists the method getIdWithoutGroup

Reply:
Oh, good! I did not see that one, thanks. Changed.